src/tests/monitoringtests.ts

```diff
@@ -217,7 +217,7 @@ describe("File Library", function() {
             fs.mkdirSync(archivePath);
         }
 
-        var laputin = compose(archivePath, new LaputinConfiguration(1234, "accurate"));
+        var laputin = compose(archivePath, new LaputinConfiguration(null));
```

Comment:
Instead of null you should pass { "identification": "accurate" } here to fix failing tests.

Also you could add some ignored file extensions like "ignoredExtensions": ["tmp"] and add at least one new test (and maybe two) to make sure ignoring works. You can use "Initial files can be found" and "When file is copied to library path, it can be found" specs as examples to make sure ignoring works both on initial hashing and when a new file is added during monitoring.